### ISSUE TITLE
mixed custom operators and raw operators in tpot

### ISSUE THREAD

**Issue by sparo-jack** [EpistasisLab/tpot#808]
how to produce following results？

exported_pipeline = make_pipeline(
    ZeroCount(),
    StackingEstimator(estimator=LGBMRegressor()),
    LassoLarsCV(normalize=True)
)

tpot_config = {
        'lightgbm.LGBMRegressor': {
        'boosting_type': ['gbdt', 'dart']
        , 'min_child_samples': [1, 5, 7, 10, 15, 20, 35, 50, 100, 200, 500, 1000]
        , 'num_leaves': [2, 4, 7, 10, 15, 20, 25, 30, 35, 40, 50, 65, 80, 100, 125, 150, 200, 250, 500]
        , 'colsample_bytree': [0.7, 0.9, 1.0]
        , 'subsample': [0.7, 0.9, 1.0]
        , 'learning_rate': [0.01, 0.05, 0.1]
        , 'n_estimators': [5, 20, 35, 50, 75, 100, 150, 200, 350, 500, 750, 1000, 1500, 2000]
        }
    }

    tpot = TPOTRegressor(generations=100, population_size=300, config_dict=tpot_config, verbosity=2)
    tpot.fit(X_train, y_train)
    tpot.export('tpot_pipeline.py')

**Comment by weixuanfu** [comment]
Below is a example:
```python
from tpot import TPOTRegressor
from tpot.config import regressor_config_dict 
regressor_config_dict ['lightgbm.LGBMRegressor'] = {
'boosting_type': ['gbdt', 'dart'],
'min_child_samples': [1, 5, 7, 10, 15, 20, 35, 50, 100, 200, 500, 1000],
'num_leaves': [2, 4, 7, 10, 15, 20, 25, 30, 35, 40, 50, 65, 80, 100, 125, 150, 200, 250, 500], 'colsample_bytree': [0.7, 0.9, 1.0],
'subsample': [0.7, 0.9, 1.0],
'learning_rate': [0.01, 0.05, 0.1],
'n_estimators': [5, 20, 35, 50, 75, 100, 150, 200, 350, 500, 750, 1000, 1500, 2000]
}

tpot = TPOTRegressor(generations=100, population_size=300, config_dict=regressor_config_dict , verbosity=2)

```
